src/convertion/safari-rules.ts

```diff
@@ -12,14 +12,17 @@ export function convertCosmetics(cosmetics: CosmeticFilter) {
   }
 
   const trigger = {};
+
   if (cosmetics.hasHostnames()) {
     const domains: string[] = [];
     const notDomains: string[] = [];
+    const punycode = require('punycode');
```

Comment:
Can this be imported at the top of the file instead? Using the typescript import syntax:
```js
import punycode from 'punycode';
```